_data/sites.json

```
@@ -15914,6 +15925,17 @@
         ]
     },
 
+    {
+        "name": "Wilson epark",
+        "url": "http://myaccount.epark.com.au/ContactUs.aspx",
+        "difficulty": "hard",
+        "notes": "No information about requesting account deletion is published on the Wilson epark website. To delete your account, you must send the company a written request. Send your request to epark either via the form on the 'Contact Us' page (linked), or by sending an email info@epark.com.au.",
```

```suggestion
        "notes": "Send your request for account deletion to epark either via the form on the 'Contact Us' page, or by sending them an email.",
```

We can make this simpler by just providing the actual instructions